syntax = "proto3";

package MvdS.ksvc;

message CryptoKey {
  string name = 1;
};
